src/module_6/database.ts

```diff
@@ -0,0 +1,60 @@
+import { Cart, Order, Product, User } from "./types"
+
+const book = {
+  id: "book",
+  title: "Book",
+  description: "Interesting book",
+  price: 200,
+}
+
+const pen = {
+  id: "pen",
+  title: "Pen",
+  description: "Cute pen",
+  price: 20,
+}
+
+const notebook = {
+  id: "notebook",
+  title: "Noteboook",
+  description: "Usefull notebook",
+  price: 50,
+}
+
+const user1: User = {
+  id: "sadmin",
```

Comment:
According to Swagger, the id must be a UUID, so the request is going to have an UUID in the header. Use JOI to validate the value of the header and check it is a valid UUID.